src/components/Container/Container.styled.ts

```diff
@@ -31,6 +31,10 @@ export const Frame = styled(Box)<{
         padding: ${pxToRem(40)} ${pxToRem(100)};
         width: ${pxToRem(600)};
         margin: 0 auto;
+        display: flex;
+        align-items: center;
+        justify-content: center;
+        flex-direction: column;
```

Comment:
u can compare `Container` with portable reputation to see the difference
but for now u can leave it as is